index.html

```diff
@@ -8,6 +8,7 @@
   </head>
 
   <body>
+    <h1>Hello</h1>
```

Comment:
🔨 @gerardlevin an h1 tag isn't the right choice, can you think of another tag that would be a better fit?